host/xtest/regression_1000.c

```diff
@@ -2570,12 +2570,14 @@ static void xtest_tee_test_1034(ADBG_Case_t *c)
 
 	res = xtest_teec_open_session(&session, &large_ta_uuid, NULL,
 				      &ret_orig);
+
```

Comment:
Extra blank line not needed

Reply:
Thanks for your time. I've corrected it.

Reply:
Good! Please add my:

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`